com.unity.render-pipelines.universal/Runtime/RenderingUtils.cs

```diff
@@ -154,27 +160,36 @@ internal class StereoConstants
         /// <param name="cmd">CommandBuffer to submit data to GPU.</param>
         /// <param name="viewMatrix">View matrix to be set. Array size is 2.</param>
         /// <param name="projectionMatrix">Projection matrix to be set.Array size is 2.</param>
+        /// <param name="cameraProjectionMatrix">Camera projection matrix to be set.Array size is 2. Does not include platform specific transformations such as depth-reverse, depth range in post-projective space and y-flip. </param>
         /// <param name="setInverseMatrices">Set this to true if you also need to set inverse camera matrices.</param>
         /// <returns>Void</c></returns>
-        internal static void SetStereoViewAndProjectionMatrices(CommandBuffer cmd, Matrix4x4[] viewMatrix, Matrix4x4[] projMatrix, bool setInverseMatrices)
+        internal static void SetStereoViewAndProjectionMatrices(CommandBuffer cmd, Matrix4x4[] viewMatrix, Matrix4x4[] projMatrix, Matrix4x4[] cameraProjMatrix, bool setInverseMatrices)
         {
             for (int i = 0; i < 2; i++)
             {
                 stereoConstants.viewProjMatrix[i] = projMatrix[i] * viewMatrix[i];
                 stereoConstants.invViewMatrix[i] = Matrix4x4.Inverse(viewMatrix[i]);
                 stereoConstants.invProjMatrix[i] = Matrix4x4.Inverse(projMatrix[i]);
                 stereoConstants.invViewProjMatrix[i] = Matrix4x4.Inverse(stereoConstants.viewProjMatrix[i]);
+                stereoConstants.invCameraProjMatrix[i] = Matrix4x4.Inverse(cameraProjMatrix[i]);
                 stereoConstants.worldSpaceCameraPos[i] = stereoConstants.invViewMatrix[i].GetColumn(3);
             }
 
             cmd.SetGlobalMatrixArray(UNITY_STEREO_MATRIX_V, viewMatrix);
             cmd.SetGlobalMatrixArray(UNITY_STEREO_MATRIX_P, projMatrix);
             cmd.SetGlobalMatrixArray(UNITY_STEREO_MATRIX_VP, stereoConstants.viewProjMatrix);
+
+            cmd.SetGlobalMatrixArray(UNITY_STEREO_CAMERA_PROJECTION, cameraProjMatrix);
+            cmd.SetGlobalMatrixArray(UNITY_STEREO_WORLD_TO_CAMERA, viewMatrix);
+            
             if (setInverseMatrices)
             {
                 cmd.SetGlobalMatrixArray(UNITY_STEREO_MATRIX_IV, stereoConstants.invViewMatrix);
                 cmd.SetGlobalMatrixArray(UNITY_STEREO_MATRIX_IP, stereoConstants.invProjMatrix);
                 cmd.SetGlobalMatrixArray(UNITY_STEREO_MATRIX_IVP, stereoConstants.invViewProjMatrix);
+
+                cmd.SetGlobalMatrixArray(UNITY_STEREO_CAMERA_INV_PROJECTION, stereoConstants.invCameraProjMatrix);
+                cmd.SetGlobalMatrixArray(UNITY_STEREO_CAMERA_TO_WORLD, stereoConstants.invViewMatrix);
```

Comment:
Similar idea, UNITY_STEREO_CAMERA_TO_WORLD and UNITY_STEREO_MATRIX_IV look the same.